test/runtests.jl

```diff
@@ -24,6 +24,11 @@ Makie.set_theme!(Theme(
         @test GeoMakie.coastlines()[1] isa GeometryBasics.LineString
     end
 
+    @testset "LineSplitting" begin
+        @test split(GeoMakie.coastlines(),"+lon_0=-160") isa Vector
+        @test split(GeoMakie.coastlines(),"+lon_0=-160")[1] isa GeometryBasics.LineString
+    end
```

Comment:
A few more tests that verifies the splitting would be useful, e.g. my suggestion from above.

Reply:
agreed

Reply:
done, I think